src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -144,6 +175,27 @@ class SlackNotificationForm extends React.Component {
                  help={get(validation, 'errors.custom_message[0]', 'Custom message to be appended below the alert title. See https://docs.graylog.org/en/latest/pages/alerts.html#data-available-to-notifications for more details.')}
                  value={config.custom_message || ''}
                  onChange={this.handleChange} />
+
+          <FormGroup>
+            <ControlLabel>Message Backlog</ControlLabel>
+            <InputGroup>
+              <InputGroup.Addon>
+                <input id="toggle_backlog_size"
+                       type="checkbox"
+                       checked={isBacklogSizeEnabled}
+                       onChange={this.toggleBacklogSize} />
+              </InputGroup.Addon>
+              <FormControl type="number"
+                           id="backlog_size"
+                           name="backlog_size"
+                           onChange={this.handleBacklogSizeChange}
+                           value={backlogSize}
+                           min="0"
+                           disabled={!isBacklogSizeEnabled} />
+            </InputGroup>
+            <HelpBlock>Limit the number of backlog messages that will be included in Slack notifications.</HelpBlock>
```

Comment:
If we have a maximum backlog size, we should also note that in the help block. Otherwise users might enter a higher number than the maximum and only will only be aware of it after the validation runs.